src/validations/sales.validation.ts

```diff
@@ -0,0 +1,24 @@
+import Joi from "joi";
+
+const salesValidation = Joi.object({
+	status: Joi.string()
+		.valid("Pending", "canceled", "delivered")
```

Comment:
As a seller i can't cancel a sale. You can correct this. Apart from that all is good

Reply:
done